src/metagit/__init__.py

```diff
@@ -388,3 +402,17 @@ def status(self) -> Tuple[Set[MetagitProject], Set[MetagitProject], Set[Path]]:
             tree = []
         untracked = {path for path in repo_path.iterdir() if path.name not in tree}
         return deleted, modified, untracked
+
+    def sync_remotes(self) -> None:
+        """
+        Create remotes in each tracked project from remotes in the Metagit repository.
+
+        InvalidRepoError is raised if self.metagit_dir does not refer to a valid repo.
+        """
+        remotes = self._git_repo().remotes
+        for project in self.projects():
+            for remote in remotes:
+                project.set_remote(
+                    remote.name,
+                    remote.url.replace(self.METAGIT_DIR_NAME, project.path.name),
```

Comment:
``` python
>>> "user@user.metagit.custom.tld:somename/.metagit.git".replace(".metagit", "foo")
'user@userfoo.custom.tld:somename/foo.git'
>>> "foo".join("user@user.metagit.custom.tld:somename/.metagit.git".rsplit(".metagit", 1))
'user@user.metagit.custom.tld:somename/foo.git'
```
```suggestion
                    project.path.name.join(remote.url.rsplit(self.METAGIT_DIR_NAME, 1))
```
https://stackoverflow.com/a/9943875/822302